### DISCUSSION TITLE
I don't have the contains faces options anymore

### DISCUSSION THREAD

**Discussion by Aksanya** [TheLastBen/fast-stable-diffusion#611] [Q&A]
Will this make my model worse? I've tried twice and I get quite deformed faces, near the ideal result but kinda glitchy, and im a confused potato

Edit: Ok I can still change it to Yes on the code. But still don't know why I get glitchy images, my 1st model was great and I wanted to finetune a 2nd one with the images of the 1st, but can't seem to replicate the quality

**Comment by TheLastBen** [comment]
if you want to use "Contains_faces" set it to "Male", "Female" or "Both". But they do not improve the result, if you want good results, keep the text encoder below 50%
